http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/AppRoutes.scala

```diff
@@ -197,15 +196,6 @@ object AppRoutes {
       } yield CreateAppRequest(c, a.getOrElse(AppType.Galaxy), d, l.getOrElse(Map.empty), cv.getOrElse(Map.empty))
     }
 
-  implicit val nameKeyDecoder: KeyDecoder[ServiceName] = KeyDecoder.decodeKeyString.map(ServiceName.apply)
-  implicit val getAppDecoder: Decoder[GetAppResponse] =
```

Comment:
Decoding GET responses and encoding POST requests are really only needed for tests, so I put them in `AppRoutesTestJsonCodec`